### ISSUE TITLE
Get frame by position from WebP and show in SimpleDraweeView

### ISSUE THREAD

**Issue by eresid** [facebook/fresco#553]
How do it?

**Comment by plamenko** [comment]
This is not currently supported out of the box.

**Comment by oprisnik** [comment]
We've released Fresco v1.4.0, which includes a new implementation for animated WebPs.

You can create a Drawee controller with a listener and then specify the given frame with something like this:

```
Fresco.newDraweeControllerBuilder()
  .setImageRequest(imageRequest)
  .setControllerListener(
          new BaseControllerListener() {
              @Override
              public void onFinalImageSet(String id, @Nullable Object imageInfo, @Nullable Animatable animatable) {
                  if (animatable != null) {
                      AnimatedDrawable2 animatedDrawable = (AnimatedDrawable2) animatable;
                      animatedDrawable.jumpToFrame(targetFrameNumber);
                  }
              }
          })
  .build());
```
